v2/googlecloud-to-googlecloud/src/test/java/com/google/cloud/teleport/v2/templates/PubSubToBigQueryIT.java

```diff
@@ -54,7 +54,7 @@
 
 /** Integration test for {@link PubSubToBigQuery} Flex template. */
 @Category(TemplateIntegrationTest.class)
-@TemplateIntegrationTest(PubSubToBigQuery.class)
+@TemplateIntegrationTest(PubSubToBigQueryAuto.class)
```

Comment:
This will remove coverage from PubSubToBigQuery and we still have it.

I'm curious if the test still work if we create a `PubSubToBigQueryAutoIT` and extend `PubSubToBigQueryIT` with the new annotation `@TemplateIntegrationTest(PubSubToBigQueryAuto.class)`?

Reply:
I was about to point this out, I had this in mind to change it back when I pushed this on Friday. But yeah, I can try creating a new test by extending the existing one

Reply:
Done